README.md

```diff
@@ -1,6 +1,59 @@
 backbone-redux
 =========================
 
+> ###Disclaimer   
+This package is extracted from [Redbooth](https://redbooth.com) to be opensourced and announced on FutureJS 2015.  
+So, the docs are not yet ready and API is still rough. But we have a huge intention to eventually make everything   eventually right.
+
 Easy way to keep your backbone collections and redux store in sync.
 
-Documentation, as always, is not yet ready. See `examples` folder.
+Creates reducers and listeners for your backbone collections and fires action creators on every collection change.
+
+### Why?
+
+* You can start migrating your apps from backbone to react+redux in no time (big blogpost about our experience is coming)
+* No need to worry about migrated/legacy parts of your app being out of sync, because both are using single source of truth
+* No boilerplate
+* You can hide all new concepts like `reducers`, `stores` and `purity` from other devs to avoid brain overloading.
```

Comment:
add action creators too. Also, I would add: "Those are just implementation details to make React performant".